app/controllers/session_controller.rb

```diff
@@ -231,6 +232,49 @@ def create
     (user.active && user.email_confirmed?) ? login(user) : not_activated(user)
   end
 
+  def email_login_page
```

Comment:
This seems clear as we don't need to scan the code to understand different path.